change_notes/2022-02-23-a13-5-2-fix-reported-fp-for-a13-5-2.md

```diff
@@ -0,0 +1 @@
+- Address a false positive on `a13-5-2` where lambda expressions with empty captures were being flagged as having a non-compliant conversion operator.
```

Comment:
```suggestion
- `A13-5-2` - address a false positive where lambda expressions with empty captures were being flagged as having a non-compliant conversion operator.
```

Reply:
@jeongsoolee09 I think just this item to address, and then we can get this merged.